components/net_traits/response.rs

```diff
@@ -13,6 +13,9 @@ use servo_arc::Arc;
 use servo_url::ServoUrl;
 use std::sync::atomic::AtomicBool;
 use std::sync::Mutex;
+// Grab the type of Std Arc that is used in FetchContext so that ResourceTiming can be passed freely between response
+// and FetchContext
+use std::sync::Arc as FetchContextArc;
```

Comment:
I don't see why this needs a special comment or name. How does this help over referring to Arc directly?

Reply:
Just putting down notes from a conversation I had with @jdm here.

- FetchContextArc is necessary because FetchContext uses a std::sync::Arc instead of servo_arc::Arc
- A better fix is to move FetchContext over to servo_arc::Arc since it saves memory anyways